src/zerg/CombatZerg.ts

```diff
@@ -161,6 +161,16 @@ export class CombatZerg extends Zerg {
 	 * Automatically ranged-attack the best creep in range
 	 */
 	autoRanged(possibleTargets = this.room.hostiles, allowMassAttack = true) {
+		let nearbyHostiles = _.filter(this.room.dangerousHostiles, c => this.pos.inRangeToXY(c.pos.x, c.pos.y, 2));
+		if(nearbyHostiles.length > 1 && !this.inRampart) {
+			this.rangedMassAttack();
+			return this.kite();
```

Comment:
`autoRanged` should never call movement commands. hydralisks could use better kiting but it should not go in the `autoMelee/Ranged/Heal` methods